apiserver/facades/client/usermanager/usermanager.go

```diff
@@ -114,6 +114,8 @@ func (api *UserManagerAPI) addOneUser(ctx context.Context, arg params.AddUser) p
 	var activationKey []byte
 	var err error
 
+	userTag := names.NewUserTag(arg.Username)
```

Comment:
This shouldn't change, because `NewLocalUserTag` implicitly enforces that the user is local. External users can not be added via the user manager.

Reply:
It does that by panicking, so ideally we would have a better way to enforce it than this, but I see your point.